finance-api/api-reference.md

```diff
@@ -0,0 +1,282 @@
+</details>
+
+
+### Responses
+<details open="true">
+<summary>HTTP status codes</summary>
+
+|Status code |Description
+|:---- |:----
+|200 OK |Returned when the payout information is successfully fetched.  
+|403 Forbidden |Returned when you are not authorised to fetch the payout information for the account.
+|400 Bad Request |Returned when a required parameter is missing or in a wrong format in the request.  
+</details>
+
+<details open="true">
+<summary>Response attributes</summary>
+
+|Name |Type |Description
+|:---- |:---- |:----
+|data |object|Information about the account payout 
+|totalBalance |integer |The account balance. It can be negative. For example, `-300`.
+|currencyId |string |The currency of the account balance. For example, `SEK`. 
+|nextPayoutAmount |integer |The amount of money that is in the `liquid` status and is ready to be paid out to the merchant.  
+|discountRemaining |integer |The amount of discounts that remains in merchant's vouchers.  
+|periodicity |string |The interval between each payout. It can be `DAILY`, `WEEKLY`, and `MONTHLY`.   
+</details>
+
+
+## Examples
+
+### Fetch balance for a liquid account
+The following example fetches balance from Zettle liquid account of the merchant that has `organizationUuid` as `18071066-bfc7-11eb-8529-0242ac130003`. As `at` is not specified for a time point, by default, it fetches the balance that is available at the moment of fecthing.
+
+Request
+```
+GET/organizations/18071066-bfc7-11eb-8529-0242ac130003/accounts/liquid/balance
+```
+Response
+```json
+{
+    "data": {
+        "totalBalance": 195,
+        "currencyId": "GBP"
+    }
+}
+```
+
+### Fetch transactions for a liquid account
+The following example fetches all transactions from Zettle liquid account of the merchant that has `organizationUuid` as `18071066-bfc7-11eb-8529-0242ac130003` from 1 January, 2021 to 7 June, 2021.
+
+Request
+```
+GET/organizations/18071066-bfc7-11eb-8529-0242ac130003/accounts/liquid/transactions?start=2021-01-01&end=2021-06-07
+```
+Response
+```json
+{
+    "data": [
+        {
+            "timestamp": "2021-04-06T22:37:37.621+0000",
+            "amount": 100,
+            "originatorTransactionType": "CARD_PAYMENT",
+            "originatingTransactionUuid": "aefbced2-9728-11eb-aa46-1cae508bb7b5"
+        },
+        ...
+        {
+            "timestamp": "2021-04-06T16:00:20.127+0000",
+            "amount": 200,
+            "originatorTransactionType": "CARD_PAYMENT",
+            "originatingTransactionUuid": "473778b2-96f0-11eb-bd49-31909a42aea2"
+        },
+    ]
+}
+```
+
+### Fetch payout information on a specific day
+The following example fetches all payout information from Zettle liquid account of the merchant that has `organizationUuid` as `18071066-bfc7-11eb-8529-0242ac130003` on 7 June, 2021.
```

Comment:
Same here, we shouldn't overcomplicate with orgUuid. It can confuse the developer as we already see it happening.